webvtt_internal_test.go

```diff
@@ -107,7 +107,8 @@ func TestTimestampMap(t *testing.T) {
 		},
 	} {
 		t.Run(strconv.Itoa(i), func(t *testing.T) {
-			offset, err := parseTimestampMapWebVTT(c.line)
+			timestampMap, err := parseTimestampMapWebVTT(c.line)
+			offset := timestampMap.Offset()
```

Comment:
Can you merge the 2 lines into `assert.Equal(t, c.expectedOffset, timestampMap.Offset())`?